docs/source/operations/projections/lcc.rst

```diff
@@ -65,9 +65,7 @@ Parameters
 
 .. include:: ../options/y_0.rst
 
-.. include:: ../options/k_0.rst
-
-.. option:: +scale=<value>
+.. option:: +k_0=<value>
```

Comment:
The other use of k_0 should be described. In LCC_1SP, this represents the "Scale factor at natural origin" parameter .

Reply:
OK. Is this definition acceptable?

    This parameter can represent two different values depending on the
    form of the projection. In LCC 1SP it determines the scale factor
    at natural origin. In LCC 2SP Michigan it determines the ellipsoid
    scale factor.

Reply:
Done.